src/ui/dashboardpage.h

```diff
@@ -30,9 +30,16 @@ class DashboardPage {
         Layout
     };
     void draw(App *app, Dashboard *Dashboard, Mode mode = Mode::View) noexcept;
+    void drawPlots(App *app, DigitizerUi::DashboardPage::Mode mode, Dashboard *dashboard);
+    void drawGrid(float w, float h);
     void drawLegend(App *app, Dashboard *dashboard, const Mode &mode) noexcept;
+
+private:
+    static bool plotButton(App *app, std::string_view glyph, std::string_view tooltip = {});
+
+private:
+    GridLayout plot_layout;
+
 };
 
 } // namespace DigitizerUi
-
-#endif
```

Comment:
usually a nice style is to echo the include lock-guard name, ie. `// DASHBOARDPAGE_H` in this case